resources/server_install.rb

```diff
@@ -30,7 +30,7 @@
 property :exec_logdir, String, default: lazy { "#{basedir}/logs" }
 property :extra_wait, Integer, default: 120
 property :framework_properties, Hash, default: {}
-property :grails_port, Integer, default: lazy { use_ssl ? 443 : 80 }
+property :grails_port, Integer, default: lazy { use_ssl ? 443 : 4440 }
```

Comment:
Not a rundeck user but why port 4440 instead of 80?

Reply:
That is the default port. The port 80 was causing the ruby block to error. I assume it worked at some point in time but now it was just causing errors.